workflow/setup_xml.py

```diff
@@ -63,6 +89,13 @@ def main(*argv):
 
     check_expdir(user_inputs.expdir, base['EXPDIR'])
 
+    # Check if "HOMEDIR","STMP","PTMP" dirrctories are writable
+    dirKeys = ["HOMEDIR","STMP","PTMP"]
```

Comment:
Pynorms would like spaces after the commas here:
```suggestion
    dirKeys = ["HOMEDIR", "STMP", "PTMP"]
```

Reply:
@DavidHuber-NOAA 
I agree with your suggestion. I have added a space between the elements.
Thanks.